lib/puppet/functions/stdlib/has_value.rb

```diff
@@ -0,0 +1,23 @@
+# Returns whether or not a value is present in a hash
+#
+# Note that this is a relatively expensive operation versus a key lookup
+#
```

Comment:
Should'nt this information be in / be copied to the `README.md` file?

Reply:
added.  thanks.  btw, henrik noted that we can just do:

```
'value' in $hash.values
```

to accomplish this, so maybe we don't need it.  I'm not sure if that discussion should be here or in the ticket.  The requestor was testing that method.